apps/condo/domains/organization/integrations/sbbol/SbbolSecretStorage.js

```diff
@@ -0,0 +1,107 @@
+const dayjs = require('dayjs')
+const conf = require('@condo/config')
+const { getRedisClient } = require('@condo/keystone/redis')
+const { getLogger } = require('@condo/keystone/logging')
+
+const SBBOL_AUTH_CONFIG = conf.SBBOL_AUTH_CONFIG ? JSON.parse(conf.SBBOL_AUTH_CONFIG) : {}
+
+// All keys belonging to SBBOL integration will have this prefix in their names
+const SBBOL_REDIS_KEY_PREFIX = 'SBBOL'
+
+// Real TTL is 180 days, but we need to update it earlier
+const REFRESH_TOKEN_TTL_DAYS = 30
+
+const logger = getLogger('sbbol/SbbolSecretStorage')
+
+/**
+ * Replaces `TokenSet` schema for storage of secrets for SBBOL API
+ */
+class SbbolSecretStorage {
```

Comment:
too match public methods. Could you please make some of them private? (by add `_`)

Reply:
I've tried to use private methods initially but something was wrong during build time.

Now I've tried to make it again and got an error:

> Class private methods are not enabled. Please add `@babel/plugin-proposal-private-methods` to your configuration.

This was the reason, that I've not used private methods.

Marking with `_` is a C++ way ;)

Reply:
@pahaz, take a look at [refactor(condo): DOMA-4226 added `@babel/plugin-proposal-private-methods` plugin to be able to use 'private' class members syntactic sugar](https://github.com/open-condo-software/condo/pull/2439/commits/e7506e7b9f5ee71c11d1b5483c785d6e54424c9b), — now we can use 'private' class members with syntactic sugar.